src/enclave/enclave_init.c

```diff
@@ -90,6 +91,8 @@ static void init_wireguard()
 
 static int startmain(void* args)
 {
+    __init_libc(sgxlkl_enclave_state.elf64_stack.envp,
```

Comment:
This can't be moved any further than this right now. `lkl_start_init` does some `getenv` calls which requires libc to be initialized.

Reply:
Don't worry. My feature.split branch will re-organize this after check in.